lib/ok.ex

```diff
@@ -526,4 +526,28 @@ defmodule OK do
       unquote(bind_match(rest) || tmp)
     end
   end
+
+  @doc """
+  Map a function over a list. This function must return a tagged result.
+  If all of the resulting values are tagged :ok, then it returns a list tagged with :ok.
+  If one or more of the resulting values are tagged :error, it returns the first error.
+
+  ## Examples
+
+  iex> OK.map_all([[1, 1], [2, 2], [3, 3]], &safe_div/1)
```

Comment:
Can we make this `OK.map_all(1..3, &save_div(6, &1)` I think the lists of lists might confuse the issue.